bin/pipeline_result_processor/prp/models/base.py

```diff
@@ -0,0 +1,13 @@
+"""Generic database objects of which several other models are based on."""
+from datetime import datetime
+
+from pydantic import BaseConfig, BaseModel, Field
+
+
+class RWModel(BaseModel):
+    """Base model for read/ write operations"""
+
+    class Config(BaseConfig):
```

Comment:
The class is very limited in scope but has a very generic name. Consider using self variables directly in RWModel

Reply:
Thats how pydantic wants you to configure the classes. See https://pydantic-docs.helpmanual.io/usage/model_config/